src/blocks/accordion/styles/style.scss

```diff
@@ -14,3 +14,20 @@ _:-webkit-full-screen,
 .wp-block-coblocks-accordion summary {
 	display: block;
 }
+
+.accordion-icon-container {
+	& > .dashicons {
+		align-items: center;
+		display: flex;
+		flex-direction: column;
+		font-size: 40px;
+		justify-content: center;
+		left: 5px;
+		position: absolute;
+		top: 20px;
```

Comment:
When testing it in local, top: 17px instead of 20px gives a more vertically centered caret. What do you think?

Reply:
Spacing is part of the experience on this that makes me hesitant to merge. We solve part of the issue by adding a caret but we fail to deliver a consistent experience, which is really the reason for this enhancement. Say, for example, the accordion title is two lines or if the title text line-height is unexpectedly large or small. In those cases, we have an offset caret.

![image](https://user-images.githubusercontent.com/30462574/121421842-7215c600-c923-11eb-9c3a-25855dd64228.png)

![image](https://user-images.githubusercontent.com/30462574/121422032-a25d6480-c923-11eb-9b79-f942984c9e5e.png)